File: src/main/java/com/maxdev/warehouse/models/Usercard.java
package com.maxdev.warehouse.models;

import jakarta.persistence.*;

import java.sql.Timestamp;

@Entity
@Table(name="usercards")
public class Usercard {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private String lastname;
    private String firstname;
    private Timestamp birthday;
    private String roles;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }

    public String getFirstname() {
        return firstname;
    }

    public void setFirstname(String firstname) {
        this.firstname = firstname;
    }

    public Timestamp getBirthday() {
        return birthday;
    }

    public void setBirthday(Timestamp birthday) {
        this.birthday = birthday;
    }

    public String getRoles() {
        return roles;
    }

    public void setRoles(String roles) {
        this.roles = roles;
    }
}

File: warehouse-app/src/services/openapi/models/Usercard.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

export type Usercard = {
    id?: number;
    lastname?: string;
    firstname?: string;
    birthday?: string;
    roles?: string;
};
export function usercardToString(el:Usercard){
    return el.id + " : " + el.lastname + " : " + el.firstname;
}
